lib/controller/page/type/page.summary/page.summary.controller.js

```diff
@@ -160,15 +164,18 @@ SummaryController.attachEmailSubmissions = async (submissionId, userData, submis
   }
 }
 
-SummaryController.attachJsonSubmission = async (submissions) => {
+SummaryController.attachJsonSubmission = async (submissions, userData) => {
   if (SERVICE_OUTPUT_JSON_ENDPOINT === undefined) {
     return
   }
 
+  const callbackUrl = `http://${SERVICE_SLUG}.formbuilder-services-${PLATFORM_ENV}-${DEPLOYMENT_ENV}:${PORT}/api/submitter/json/default/${userData.getUserId()}.json`
```

Comment:
Wouldn't this be better to pass the `http://${SERVICE_SLUG}.formbuilder-services-${PLATFORM_ENV}-${DEPLOYMENT_ENV}` part in as an env var?

Reply:
I've now extracted this to constants